newIDE/app/src/stories/componentStories/UserChip.stories.js

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
I know this PR is already long, but should we move the existing `UserPublicProfileChip` to this file too?
I think they're quite similar and it'd be good to have them next to each other

Reply:
Oh good idea, will do!